physics/docs/pdftxt/SRW_all_schemes_list.txt

```diff
@@ -16,7 +16,6 @@ The UFS-SRW App. assembles the parameterizations in suites.
  - \subpage GFS_NOAH 
  - \subpage RUCLSM 
  - \subpage NoahMP 
```

Comment:
Is there a suite that uses NoahMP? May need to remove this line

Reply:
There are suites in SRW (e.g. `FV3_GFS_v17_p8`) that make use of it, but none of the supported suites for UFS v3.0 do. This PR is technically to the ufs/dev branch and not the release branch, so it can be removed when this documentation is moved to the release branch.